website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,12 +24,15 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
+                    limit_recipient = rec.env['ir.default'].get('res.config.settings', 'email_recipient_limit')
+                    number_of_loop = round(len(partners)/limit_recipient)
```

Comment:
@TimLai125  And this is a variable, I used to store how many time we need to loop based on our setting.

Reply:
The calculation of the looping is not correct. When the length of the list is 12, and the limit is 10, we need to loop 2 times but the calulcation above will only give 1.